src/middlewares/handlelog.js

```diff
@@ -3,7 +3,6 @@ const { v4: uuid } = require('uuid');
 const log=require("../models/log");
```

Comment:
Keep your code consistent. Either use the semicolon in every statement or don't at all.